src/components/Card/Card.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./Card.css";
+
+export default function Card() {
+  return (
+    <section className="card-image-container">
+      <img className="card-image" src="/Assets/images/female-avatar.png" alt="Avatar" />
+      <div className="card-text">
+        <span className="card-name">Anna S.</span>
```

Comment:
Are `<spans>` best practice here for html?
I'm not saying they're not, and I'm not sure what you'd replace them with other than `<div>`s. Just mentioning it because I'm sure I've seen people say in a few places to avoid spans unless you can't